.github/workflows/codeql.yml

```diff
@@ -47,7 +47,7 @@ jobs:
 
       # Initializes the CodeQL tools for scanning.
       - name: Initialize CodeQL
-        uses: github/codeql-action/init@v2
+        uses: github/codeql-action/init@v3
```

Comment:
hmm v3 tags exist but a v3 release doesn't. Going to hold off until an official release. 

Ref: 
- https://github.com/github/codeql-action/releases 
- https://github.com/github/codeql-action/tags